### ISSUE TITLE
Highlight annotated code using the same syntax setting as the original file.

### ISSUE THREAD

**Issue by kavehv** [ryu1kn/vscode-annotator#1]
This would be great to have.

**Comment by ryu1kn** [comment]
Yeah, this would be awesome, but as the annotation view is just a html and not a `TextDocument` I don't think I can easily make it. For now, I would just hope vscode someday introduce a function to let us have a left column in each textdocument. Then I would stop using html view, simply use a textdocument and feed the left column with the annotation.

**Comment by agauniyal** [comment]
https://github.com/Microsoft/monaco-editor 😆

**Comment by ryu1kn** [comment]
Haha. Well it seems there is at least a way though it would be a bit overkill :p

**Comment by ryu1kn** [comment]
> For now, I would just hope vscode someday introduce a function to let us have a left column in each textdocument. Then I would stop using html view, simply use a textdocument and feed the left column with the annotation.

This appears to be the feature request I also wanted: https://github.com/Microsoft/vscode/issues/9561

**Comment by razvanphp** [comment]
@ryu1kn Your feature request seems to be solved: Microsoft/vscode#9561

Does that mean that you now have everything you need in the API to use a `TextDocument`?

**Comment by ryu1kn** [comment]
The issue has been closed with the new feature for just switching the way to display line numbers; and that's not enough for Annotator to put annotations into gutter.

**Comment by santeriv** [comment]
Thanks for getting vscode slightly better with this extension.

+1 (this issue) , to be honest for me this is a must-have, not just could-have. Blame other IDEs out there :)

**Comment by OnkelTem** [comment]
Is it still impossible to get it highlighted?
